src/Compilers/Server/VBCSCompilerTests/DesktopBuildClientTests.cs

```diff
@@ -18,42 +18,6 @@ namespace Microsoft.CodeAnalysis.CompilerServer.UnitTests
 {
     public class DesktopBuildClientTests : TestBase
     {
-        private sealed class ServerInfo
-        {
-            public readonly string ClientDirectory;
-            public readonly string SdkDirectory;
-            public readonly List<Task> Servers = new List<Task>();
-            public int CreateCount;
-
-            internal ServerInfo()
-            {
-                ClientDirectory = Path.GetDirectoryName(typeof(DesktopBuildClientTests).Assembly.Location);
-                SdkDirectory = RuntimeEnvironment.GetRuntimeDirectory();
-            }
-
-            public bool TryCreateServer(string pipeName)
-            {
-                CreateCount++;
-                Servers.Add(CreateServerCore(pipeName));
-                return true;
-            }
-
-            private Task CreateServerCore(string pipeName)
-            {
-                Action action = () =>
-                {
-                    var compilerServerHost = new DesktopCompilerServerHost(ClientDirectory, SdkDirectory);
-                    var clientConnectionHost = new NamedPipeClientConnectionHost(compilerServerHost, pipeName);
-                    var mutexName = $"{pipeName}.server";
-                    VBCSCompiler.Run(mutexName, clientConnectionHost, TimeSpan.FromSeconds(3));
-                };
-
-                var task = new Task(action, TaskCreationOptions.LongRunning);
-                task.Start(TaskScheduler.Default);
-                return task;
-            }
-        }
-
         private sealed class TestableDesktopBuildClient : DesktopBuildClient
         {
             private readonly string _pipeName;
```

Comment:
Agree.  I've renamed, locally, all uses of `pipeName` to `sessionKey` when it refers to the abstract concept.  The term `pipeName` is still in use in places where it specifically refers to a named pipe.